packages/web/index.js

```diff
@@ -93,18 +98,6 @@ module.exports = (neutrino, opts = {}) => {
     };
   }
 
-  if (typeof options.devServer.proxy === 'string') {
-    options.devServer.proxy = {
-      '**': {
-        target: options.devServer.proxy,
-        changeOrigin: true,
-        headers: {
-          Forwarded: 'by=_webpack-dev-server'
-        }
-      }
-    };
-  }
-
```

Comment:
Should we include the `headers` bit in our example? If not here, perhaps a page in the docs for "Using Neutrino with Existing SSR Architecture" would make sense.

Reply:
I think we could easily write several pages on how to set up SSR. I'm pretty sure html-webpack-plugin doesn't fully work with out of the box and needs some experimental options enabled.

I think this example is perhaps best aimed at the more common case of "I have a standard SPA but it uses a backend and I want to proxy to it in development". (Not that it precludes adding a more advanced SSR guide somewhere else in the future)

Reply:
Also, perhaps an SSR guide is best suited as an extra page in webpack's docs? Ultimately Neutrino is not meant to be a replacement for all of webpack/...'s documentation, but a very thin layer of abstraction that assists with configuration generation, and that makes it easy to mentally map between a webpack guide and how to add that configuration via `.neutrinorc.js` or a custom Neutrino preset.

Reply:
#1177 :)

Reply:
> I think we could easily write several pages on how to set up SSR

I underestimated...!
https://ssr.vuejs.org/

Reply:
Also: it's really 2 different things if you're talking about SSR with React/Vue vs. using webpack/neutrino for "assets", and something else for SSR of html (PHP, Ruby, etc)

Reply:
Yeah true